code/identity/src/main/java/com/adobe/marketing/mobile/identity/IdentityExtension.java

```diff
@@ -860,31 +844,22 @@ boolean handleSyncIdentifiers(
             return false;
         }
 
-        // org id is a requirement.
-        // Use what's in current config shared state. if that's missing, check latest config.
-        // if latest config doesn't have org id either, IdentityExtension can't proceed.
-        ConfigurationSharedStateIdentity currentEventValidConfig;
-
-        if (!StringUtils.isNullOrEmpty(configSharedState.orgID)) {
-            currentEventValidConfig = configSharedState;
-        } else {
-            if (latestValidConfig != null) {
-                currentEventValidConfig = latestValidConfig;
-            } else {
-                // can't process this event. return false to break execution loop
-                Log.debug(
-                        IdentityConstants.LOG_TAG,
-                        LOG_SOURCE,
-                        "handleSyncIdentifiers : Unable to process sync identifiers request as the"
-                                + " configuration did not contain a valid Experience Cloud"
-                                + " organization ID. Will attempt to process event when a valid"
-                                + " configuration is received.");
-                return false;
-            }
+        if (latestValidConfig == null) {
+            // sanity check, should never get here as latestValidConfig set from
```

Comment:
nit: this should not have any impact on the flow, but for consistency with iOS can we do this sanity check first thing in this method?